styles.css

```diff
@@ -1,15 +1,1236 @@
+    display: flex;
+    margin: 0 20px 10px 20px;
+    justify-content: space-between
+}
+.quantity{
+    display: flex;
+    border: solid 1px #ccc;
+    border-radius: 5px;
+    /* background-color: #7289d9; */
+     
+}
+.plus-minus{
+    width:25px;
+    height: 20px;
+    border: none;
+    font-size: 120%;
+    text-align: center;
+    color: rgb(82, 81, 81)
+}
+.number-item{
+    width: 25px;
+    height: 20px;
+    text-align:center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.remove-item{
+    padding: 1px;
+    width:90px;
+    height: 20px;
+    color: rgb(82, 81, 81)
+}
+.remove-item:hover{
+color: black;
+border: solid 1px black;
+}
+
+
+/* courosel */
+/* Slideshow container */
+.slideshow-container {
+  max-width: 650px;
+  height: 30%;
+  /* position: relative; */
+  margin: auto;
+}
+
+/* Hide the images by default */
+.mySlides {
+  display: none;
+}
+
+/* Next & previous buttons */
+.prev, .next {
+  cursor: pointer;
+  /* position: absolute; */
+  top: 50%;
+  width: auto;
+  margin-top: -22px;
+  margin-left: 24%;
+  padding: 16px;
   color: white;
+  font-weight: bold;
+  font-size: 18px;
+  transition: 0.6s ease;
+  border-radius: 0 3px 3px 0;
+  user-select: none;
+}
+.next{
+    margin-left:40%;  
+}
+
+/* Position the "next button" to the right */
+.next {
+  right: 0;
+  border-radius: 3px 0 0 3px;
+}
+
+/* On hover, add a black background color with a little bit see-through */
+.prev:hover, .next:hover {
+  background-color: rgba(0,0,0,0.8);
+}
+
+/* Caption text */
+.text {
+  color: #f2f2f2;
+  font-size: 15px;
+  padding: 8px 12px;
+  /* position: absolute; */
+  bottom: 8px;
+  width: 100%;
+  text-align: center;
+}
+
+/* Number text (1/3 etc) */
+.numbertext {
+  color: #f2f2f2;
+  font-size: 12px;
+  padding: 8px 12px;
+  /* position: absolute; */
+  top: 0;
+}
+
+/* The dots/bullets/indicators */
+.dot {
+  cursor: pointer;
+  height: 15px;
+  width: 15px;
+  margin: 0 2px;
+  background-color: #bbb;
+  border-radius: 50%;
+  display: inline-block;
+  transition: background-color 0.6s ease;
+}
+
+.active, .dot:hover {
+  background-color: #717171;
 }
 
-.container {
-  width: 90%;
-  height: auto;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
+/* Fading animation */
+/* .fade {
+  -webkit-animation-name: fade;
+  -webkit-animation-duration: 0s;
+  animation-name: fade;
+  animation-duration: 0s;
 }
+
+@-webkit-keyframes fade {
+  from {opacity: .4}
+  to {opacity: 1}
+}
+
+@keyframes fade {
+  from {opacity: .4}
+  to {opacity: 1}
+} */
+
+/* footer */
+footer{
+    background-color: black;
+    color: #f4f4f4;
+    display: flex;
+    flex-direction: column
+}
+.first-line{
+    
+    display: flex;
+    padding: 10px;
+    /* margin:auto; */
+    justify-content: space-evenly;
+}
+.under-stuff{
+    display: flex;
+    font-size: 80%;
+    
+}
+.copyright{
+    text-align: center;
+    font-size: 80%
+}
+
+
```

Comment:
why large space here?

Reply:
the space was to enable me locate the media query for mobile section easily 
i'd take it off